Error: Cannot use optional chaining

Multiple errors in the file Reference HTML file

```
DocCHTMLExporter/HTML/ReferenceHTML.swift:93:23: error: cannot use optional chaining on non-optional value of type 'String'
 91 |         
 92 |       case .image(let ref):
 93 |         return ref.alt?.htmlEscaped ?? ""
    |                       `- error: cannot use optional chaining on non-optional value of type 'String'
 94 | 
 95 |       case .link(let ref):
```

--- a/Sources/DocCHTMLExporter/HTML/ReferenceHTML.swift
+++ b/Sources/DocCHTMLExporter/HTML/ReferenceHTML.swift
@@ -90,7 +90,7 @@ extension DocCArchive.Reference {
         return ref.abstract.generateHTML(in: ctx)
         
       case .image(let ref):
-        return ref.alt?.htmlEscaped ?? ""
+        return ref.alt.htmlEscaped
 
       case .link(let ref):
         return ref.title.htmlEscaped
@@ -150,7 +150,7 @@ extension DocCArchive.ImageReference {
   func generateHTML(in ctx: DZRenderingContext) -> String {
     guard let variant = bestVariant(for: ctx.traits) else {
       assertionFailure("Got no image variant?")
-      return "<!-- invalid image ref -->\(alt?.htmlEscaped ?? "")"
+      return "<!-- invalid image ref -->\(alt.htmlEscaped)"
     }
     
     let media : String = {
@@ -173,7 +173,7 @@ extension DocCArchive.ImageReference {
     var ms = "<picture><source\(media) srcset='\(srcset)'>"
 
     ms += "<img"
-    if let alt = alt, !alt.isEmpty { ms += " alt='\(alt.htmlEscaped)'" }
+    if !alt.isEmpty { ms += " alt='\(alt.htmlEscaped)'" }
     ms += " srcset='\(srcset)'"
     ms += " src='\(url.htmlEscaped)'"
     if let size = variant.size { ms += " width='\(size.width)' height='auto'" }

